modules/ocf/manifests/packages/brave.pp

```diff
@@ -1,34 +1,11 @@
 class ocf::packages::brave {
-  include ocf::browser_sandbox
-
-  $browser_homepage = lookup('browser_homepage')
-
-  class { 'ocf::packages::brave::apt':
-    stage =>  first,
-  }
-
   # Remove the old version of brave
   package { 'brave':
     ensure => purged
   }
-
-  # New install instructions, per https://brave-browser.readthedocs.io/en/latest/installing-brave.html#linux
-  package { 'brave-browser':; } ->
-
-  # TODO: Change this to a global config
-  # if/when Brave adds support for it.
-  file {
-    [
-      '/etc/skel/.config/BraveSoftware/',
-      '/etc/skel/.config/BraveSoftware/Brave-Browser/',
-      '/etc/skel/.config/BraveSoftware/Brave-Browser/Default'
-    ]:
-      ensure  => directory,
-      require => File['/etc/skel/.config'],
-  } ->
-
-  file {
-    '/etc/skel/.config/BraveSoftware/Brave-Browser/Default/Preferences':
-      content => template('ocf/brave/Preferences.erb'),
```

Comment:
Make sure to delete this file `modules/ocf/templates/brave/Preferences.erb`. I don't see it in the diff.